app/src/main/java/ru/practicum/android/diploma/data/db/converter/AreaTypeConverter.kt

```diff
@@ -0,0 +1,26 @@
+package ru.practicum.android.diploma.data.db.converter
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import ru.practicum.android.diploma.domain.models.Area
+
+class AreaTypeConverter {
+
+    private val gson = Gson()
+
+    @TypeConverter
+    fun fromArea(area: Area?): String? {
+        return area?.let {
+            gson.toJson(it)
+        }
```

Comment:
```suggestion
        return gson.toJson(it)
```
он вернет или данные, или null